src/slic3r/GUI/MediaPlayCtrl.cpp

```diff
@@ -389,7 +389,7 @@ void MediaPlayCtrl::Stop(wxString const &msg)
     }
 
     auto tunnel = m_url.empty() ? "" : into_u8(wxURI(m_url).GetPath()).substr(1);
-    if (auto n = tunnel.find_first_of('/_'); n != std::string::npos)
```

Comment:
I don't even know c++ allows this!

Reply:
Yes I was also surprised this is allowed